### ISSUE TITLE
About resume model

### ISSUE THREAD

**Issue by Stefanie927** [wentaozhu/DeepLung#97]
When I train the res18 model, I can resume the 064.ckpt, but if I want to train the dpn model, which model can I resume?

**Comment by wentaozhu** [comment]
The pretrained dpn model is in https://github.com/wentaozhu/DeepLung/tree/master/detector/dpnmodel

**Comment by FancccyRay** [comment]
I also wonder which model responding to folder 1,2,3,4...10,哪个模块对应哪个子集？

**Comment by wentaozhu** [comment]
Fold i is using the subset i as validation set.

**Comment by FancccyRay** [comment]
> Fold i is using the subset i as validation set.
I mean I want to resume your model, however, there are 10 models, like which fold i should use to validate model fd0066.ckpt?

**Comment by wentaozhu** [comment]
Fold 0
